nix/container.nix

```diff
@@ -0,0 +1,22 @@
+{ pkgs, lib, inputs ? builtins.throw "Expected an inputs argument that corresponds to flake conventions", ... }: {
```

Comment:
```suggestion
inputs: { pkgs, lib, ... }: {
```

Reply:
I deliberately avoid this so that overlays and modules can be imported directly without the rest of the flake code.

Reply:
Wouldn't this throw the exception every time the module is called without `inputs`, since it's explicitly defined?

Reply:
@deliciouslytyped ?